auth-web/src/components/auth/CreateUserProfileForm.vue

```diff
@@ -173,8 +173,8 @@ export default class CreateUserProfileForm extends Mixins(NextPageMixin) {
               case 'FAILED_ADDING_USER_IN_KEYCLOAK':
                 this.showErrorModal('Failed to add the user, please try again')
                 break
-              case 'DATA_ALREADY_EXISTS':
-                this.showErrorModal('There is already an user existing with the given username, please try with a new one')
+              case 409:
```

Comment:
The switch is checking the code, so this change is good.  Shouldn't the case above it also be changed, since it's checking the message?

Reply:
yup..will ask @nitheesh-aot the reasoning..then will change it